spec/tapioca/cli/check_shims_spec.rb

```diff
@@ -1,7 +1,7 @@
 # typed: true
 # frozen_string_literal: true
 
-require "spec_with_project"
+require "spec_helper"
```

Comment:
One problem is that we make all test always require `tapioca/internal` and all what comes with it when it's not always needed. This breaks a bit the isolation.

Reply:
Does it matter? All of the ones that were explicitly using `spec_with_project`, like this one are CLI tests that run in a separate process, no?

Reply:
For example this test will not be isolated the same way: https://github.com/Shopify/tapioca/blob/main/spec/tapioca/gemfile_spec.rb#L5

Not sure if it's okay or not. Maybe we should refactor the test itself. What do you think?

Reply:
I'm good with this. We can always create another isolated spec helper if we need it later 👍